src/prefect/tasks/notifications/email_task.py

```diff
@@ -50,6 +56,7 @@ def __init__(
         msg_plain: str = None,
         email_to_cc: str = None,
         email_to_bcc: str = None,
+        attachments: List[str] = [],
```

Comment:
We want to avoid using any [mutable default arguments](https://florimond.dev/blog/articles/2018/08/python-mutable-defaults-are-the-source-of-all-evil/). I would default this to `None`. Then, if nothing is passed, set `self.attachments` to an empty list